kubernetes/samples/scripts/create-soa-domain/docs-source/supportJDEV.md

```diff
@@ -0,0 +1,88 @@
+# Deploying SOA Composties from Oracle JDeveloper to Oracle SOA in WebLogic Kubernetes Operator Environment
+
+This documents provides steps to deploy SOA composites/applications from Oracle JDeveloper (that runs outside the Kubernetes network) to the SOA instance in WebLogic Kubernetes Operator Environment.
+
+```
+Note: Dev and Test environment only. For production you should deploy using Application Control and WLST methods.
+```
+
+## Deploy to SOA from JDeveloper
+
+To deploy SOA composites/applications from Oracle JDeveloper, Administration Server should have been configured to expose a T3 channel using the *exposeAdminT3Channel* setting when creating the domain, then the matching T3 service can be used to connect.
+
+By default when *exposeAdminT3Channel* is set WebLogic Kubernetes Operator Environment will expose NodePort for the T3 channel of the NetworkAccessPoint at 30012 (Use *t3ChannelPort* to configure port to different value).
+
+![SOA Composite Deployment Model](images/SOA_Composites_Deploy_using_Jdev.png)
+
+## Prerequisities
+
+Note: Replace entries inside <xxxx> specific to your environment
+
+NOTE  : Opening the managed server port is not as secure as the default, where the port is not exposed. If you do not want to open this port, this solution does not work for you.
+
+1.  Get the Kubernetes Cluster Master Address and verify the T3 port which will be used for creating application server connections. You can use below kubectl command to get the T3 port:
+
+        kubectl get service <domainUID>-<AdminServerName>-external -n  <namespace>-o jsonpath='{.spec.ports[0].nodePort}'
```

Comment:
The style of other user commands is to start with a $ and then use backticks, such as, `$ kubectl get service...`